cmd/minikube/cmd/start.go

```diff
@@ -1370,6 +1370,14 @@ func validateKubernetesVersion(old *config.ClusterConfig) {
 		exitIfNotForced(reason.KubernetesTooOld, "Kubernetes {{.version}} is not supported by this release of minikube", out.V{"version": nvs})
 	}
 
+	// If the version of Kubernetes has a known issue, print a warning out to the screen
+	if issue := reason.ProblematicK8sVersion(nvs); issue.Description != "" {
```

Comment:
This should return a pointer. And check if not nil

Reply:
done